pkg/scheduler/scheduler.go

```diff
@@ -203,7 +207,9 @@ func (s *Scheduler) assign(ctx context.Context, e *entry) {
 	go func() {
 		err := s.client.Update(ctx, newWorkload)
 		if err == nil {
-			log.V(2).Info("Successfully assigned capacity and resource flavors to workload")
+			msg := fmt.Sprintf("Successfully assigned %v to capacity %v", workload.Key(newWorkload), e.Capacity)
```

Comment:
```suggestion
			msg := fmt.Sprintf("Successfully assigned to capacity %v", e.Capacity)
```

The event is added for the workload, so we don't need to mention it's name again.